tests/vmi_sound_test.go

```diff
@@ -40,51 +40,37 @@ import (
 	"kubevirt.io/kubevirt/tests/testsuite"
 )
 
-var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-compute] Sound", decorators.SigCompute, func() {
-
-	var virtClient kubecli.KubevirtClient
-
-	BeforeEach(func() {
-		virtClient = kubevirt.Client()
-	})
+var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-compute] Sound", decorators.SigCompute, decorators.Conformance, func() {
 
 	Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component] A VirtualMachineInstance with default sound support", func() {
 
 		It("should create an ich9 sound device on empty model", func() {
-			vmi, err := createSoundVMI(virtClient, "test-model-empty")
+			vmi, err := createSoundVMI(kubevirt.Client())
 			Expect(err).ToNot(HaveOccurred())
 			vmi = libwait.WaitUntilVMIReady(vmi, console.LoginToCirros)
-			checkAudioDevice(vmi, "ich9")
+			err = checkICH9AudioDeviceInGues(vmi)
+			Expect(err).ToNot(HaveOccurred(), "ICH9 sound device was no found")
 		})
 	})
 
 })
 
-func createSoundVMI(virtClient kubecli.KubevirtClient, soundDevice string) (*v1.VirtualMachineInstance, error) {
+func createSoundVMI(virtClient kubecli.KubevirtClient) (*v1.VirtualMachineInstance, error) {
 	randomVmi := libvmifact.NewCirros()
```

Comment:
nit: this vmi is not really "random" in any meaningful way. Consider renaming it to `vmi`.

Reply:
I will edit this once we close the conversion above in one or another way.

Reply:
btw, you may opt to drop the `virtClient` argument too. This would let you drop another ugly global.